enclave-persistence/gradle.properties

```diff
@@ -0,0 +1,3 @@
+#
+conclaveVersion=1.2
+conclaveRepo=/Users/divyataori/Desktop/Conclave/conclave-sdk-1.2/repo
```

Comment:
We can't commit with your local path in sample. First thing anyone using this sample will have to do is modify this. What was the outcome from Rusheb and @snedamle's discussion on the conclave repo location in this file? Is it relative path?

Reply:
Updated